base_joints:
  base_x_slide_joint: 0.0
  base_y_slide_joint: 0.0
  base_z_hinge_joint: 0.0

manipulator_joints:
  joint1: 0.0
  joint2: -0.785398163397
  joint3: 0.0
  joint4: -2.35619449019
  joint5: 0.0
  joint6: 1.57079632679
  joint7: 0.785398163397

finger_joints:
  finger_joint1: 0.01
  finger_joint2: 0.01
